Makefile

```
@@ -48,6 +48,11 @@ help: ## Print this help
 	@echo
 	@$(PYTHON) -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
 
+test-rc-env: ## Replace the Beam vesion with the latest RC version
```

```suggestion
test-latest-env: ## Replace the Beam vesion with the latest version (including release candidates)
```

(if there's no active RC this will choose the latest released version, not the RC)

---

(if we change task name here, we should do so in the workflow too)

---

Make sense. Just changed this and also the workflow file. Thanks.